### PR TITLE
fix(overlap): Issue #606 - Overlap Style Missing on Added Task

### PR DESCRIPTION
**Pull request by TomSchober**
Handle task add.  Added timeout similar to data change handler. Added demo control.

### CODE REVIEW

File: src/plugins/overlap.js [hunk 1]
```diff
@@ -137,6 +137,18 @@
                             handleOverlaps(oldRow.tasks);
                         }
                     });
+
+                    api.tasks.on.add(scope, function(task) {
+                        // TODO: Mimicked functionality from api.data.on.change to defer until element creation, but not ideal.  Refactor necessary to raise 'add' event after task is fully drawn.
```
Comment (Toilal) [inline src/plugins/overlap.js]: There's a viewChange event, but I should implement viewAdd and viewRemove too for this timeout to be removed.

**Comment by TomSchober** [comment]
Not sure what tests should be added to entertain coveralls

**Comment by TomSchober** [comment]
@Toilal , what do you suggest I add to this PR for acceptance?

**Comment by Toilal** [comment]
Sorry for delay @TomSchober.

I'll merge and try to add viewAdd and viewRemove on api.tasks later. Thanks !

**Comment by TomSchober** [comment]
Not a problem.   I plan on contributing a lot more to this project.  Thanks.

**Comment by Toilal** [comment]
@TomSchober i'm happy to hear this ! If you want, we could include you in the team so you can have write access to the origin repository.

@Schweigi would you be OK ?

**Pull request merged**